docs/articles/documentation/recipes/using-page-model.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: docs
+title: Using Page Model
+permalink: /documentation/recipes/using-page-model.html
+---
+# Using Page Model
+
+[Page Model](http://martinfowler.com/bliki/PageObject.html) is a test automation pattern that allows you to create an abstraction of the tested page
+and use it in test code to refer to page elements.
+
+## Why Use Page Model
+
+Consider the following fixture with two tests: one that types and edits
+the developer name on the [example](https://devexpress.github.io/testcafe/example/) webpage and the other that
+checks check boxes in the Features section.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+test('Text typing basics', async t => {
+    await t
+        .typeText('#developer-name', 'Peter')
+        .typeText('#developer-name', 'Paker', { replace: true })
+        .typeText('#developer-name', 'r', { caretPos: 2 })
+        .expect(Selector('#developer-name').value).eql('Parker');
+});
+
+test('Click check boxes and then verify their state', async t => {
+    await t
+        .click('input[id=remote-testing]')
+        .expect(Selector('input[id=remote-testing]').checked).ok()
+        .click('input[id=reusing-js-code]')
+        .expect(Selector('input[id=reusing-js-code]').checked).ok()
+        .click('input[id=continuous-integration-embedding]')
+        .expect(Selector('input[id=continuous-integration-embedding]').checked).ok();
+});
+```
+
+Note that both tests contain excessive code.
+In the first test, the `#developer-name` CSS selector is duplicated in code each time the test refers to the input element.
+In the second test, test logic is duplicated for each check box.
+
+Page Model allows you to keep all selectors in one place -
```

Comment:
We need to reflect the fact that page markup/design may changes often, therefore abstracting "page description" into a single place allows you to not modify all tests, but just do it in the single place. It's a separation of concerns with all its benefits: you describe page representation in the PageModel and you describe how it should behave in your tests.